datasetIngestor/sendIngestCommand.go

```diff
@@ -34,112 +47,144 @@ func createOrigBlock(start int, end int, filesArray []Datafile, datasetId string
 		totalSize += filesArray[i].Size
 	}
 	// fmt.Printf("Start:%v, end:%v, totalsize:%v\n, first entry:%v\n", start, end, totalSize, filesArray[start])
-
+	
```

Comment:
extra tab

Reply:
I don't think this is wrong because this is what happens when vscode reindents the code.

Reply:
mmmm vscode does the reindent because it has a default linter. But maybe we should review later on the linter settings and agree on a linter